src/Common/TraceCollector.cpp

```diff
@@ -61,17 +62,19 @@ TraceCollector::~TraceCollector()
 
 void TraceCollector::collect(TraceType trace_type, const StackTrace & stack_trace, Int64 size)
 {
-    constexpr size_t buf_size = sizeof(char) + // TraceCollector stop flag
-        8 * sizeof(char) +                     // maximum VarUInt length for string size
```

Comment:
According to the style, comments should start with three slashes instead of two and sentences should start with Capital.